File: 8_kyu/I_love_you_a_little_a_lot_passionately_not_at_all.go
package kata

var text = []string{
	"I love you",
	"a little",
	"a lot",
	"passionately",
	"madly",
	"not at all"}

func HowMuchILoveYou(i int) string {
	if i > 0 {
		return text[(i-1)%len(text)]
	} else {
		return "error"
	}
}

File: 8_kyu/I_love_you_a_little_a_lot_passionately_not_at_all.rs
fn how_much_i_love_you(nb_petals: u16) -> &'static str {
    let index = if nb_petals > 0 {
        (nb_petals - 1) % 6
    } else {
        0
    };
    match index {
        0 => "I love you",
        1 => "a little",
        2 => "a lot",
        3 => "passionately",
        4 => "madly",
        5 => "not at all",
        _ => "",
    }
}
